src/containers/Statistics/StatisticsCard.tsx

```diff
@@ -3,11 +3,12 @@ import React from 'react'
 import { useSelector } from 'react-redux'
 
 import statsSelector from '@selectors/stats'
-import { getCollateralValue, getSyntheticsStructure } from '@selectors/exchange'
+import { getCollateralValue, getSyntheticsStructure, getSwaplineCollateralBalance } from '@selectors/exchange'
 
 export const StatisticsCard: React.FC = () => {
   const statsData = useSelector(statsSelector.last24)
   const synthetics = useSelector(getSyntheticsStructure)
+  const USDBalance = useSelector(getSwaplineCollateralBalance)
```

Comment:
maybe swaplineCollateralValue instaed of USDBalance?

Reply:
okay, make sense